app/client/src/routes/home.tsx

```diff
@@ -1,16 +1,511 @@
+import { useCallback, useEffect, useReducer, useRef, useState } from 'react';
+import Select from 'react-select';
 // components
-import { Loading } from "components/loading";
+import Alert from 'components/alert';
+import Button from 'components/button';
```

Comment:
```suggestion
```

Part of a comment above. I don't think this component is necessary at this time.